### DISCUSSION TITLE
Browsers compatibility

### DISCUSSION THREAD

**Discussion by atalargo** [vuejs/petite-vue#55] [General]
First, great project and it seems very cool (don't try it yet). But I have one question: What's about the compatibility? 
- "template" tag could have issues in some browsers?
- "let", "const", "import" keywords could have issues in some browsers?
- 'type="module"' for 'script' tag is managed everywhere?
- other compatibility issues.

FRD

**Comment by mingjiezhou** [comment]
刚准备用在企业微信应用里, 做了个测试 发现 IE和企业微信的内核都不兼容啊

**Comment by niutech** [comment]
The [unpkg bundle](https://unpkg.com/petite-vue@0.2.3/dist/petite-vue.iife.js) uses [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), so it won't work in IE. However, you can use any bundler (such as Vite, Webpack, Parcel, Rollup) with Babel to build your app bundle with ES5 compatibility.
